### PR TITLE
add bgcolor and gridbox color on __init__ file

### PR DESCRIPTION
**Pull request by PennyQ**
Shall we move it to the `glue.config`? @astrofrog

### CODE REVIEW

File: glue_vispy_viewers/__init__.py [hunk 1]
```diff
@@ -1 +1,3 @@
-from .version import __version__
\ No newline at end of file
+from .version import __version__
+BACKGROUND_COLOR = 'black'
+GRIDBOX_COLOR = 'white'
```
Comment (astrofrog) [inline glue_vispy_viewers/__init__.py]: I would just call this FOREGROUND_COLOR since it could apply to axes, labels, etc.

**Comment by astrofrog** [comment]
@PennyQ - once we move it to the core, we should also make sure we implement it for the other 2D viewers (that might take a little while to figure out)

**Comment by PennyQ** [comment]
@astrofrog Got it, thanks and here we go :)

**Comment by astrofrog** [comment]
@PennyQ - thanks!

**Comment by astrofrog** [comment]
I'll add an entry in `CHANGES.md`, but make sure you include one in future pull requests! :)

**Comment by PennyQ** [comment]
@astrofrog Thanks for reminding, I will keep in mind ;)

**Pull request merged**